src/components/CardList.tsx

```diff
@@ -26,9 +26,9 @@ const CardList = ({ children }: Props) => {
     <Flex w="full" justifyContent="center" alignItems="center">
```

Comment:
I think we need a lil extra on the horizontal margins here:
![image](https://user-images.githubusercontent.com/8410924/114497313-d4866a80-9bef-11eb-9436-e7dbfdf7f2b3.png)

Reply:
Also I think the `HStack` that contains the actual cards should have `alignItems="start"`